Makefile

```diff
@@ -1,36 +1,40 @@
 # get the basename of the containing directory
 # this will be used to name othe output document
 BUILD_DIR := $(shell basename $(abspath $(dir $(lastword $(MAKEFILE_LIST)))))
-PDF_FILE?=mypdf
+BUILD_DIR ?= presentation
 
-all: ${PDF_FILE}.pdf
+all: ${BUILD_DIR}.pdf
 
-view:
-	atom ${PDF_FILE}.pdf
-
-sview:
-	gnome-open ${PDF_FILE}.pdf 2>/dev/null
+view: ${BUILD_DIR}.pdf
+	[ "$(shell uname)" = "Darwin" ] && open ${BUILD_DIR}.pdf \
+		|| xdg-open ${BUILD_DIR}.pdf 2>/dev/null
 
 mmore500-presentation-template-master-latest.simg:
 	singularity pull shub://mmore500/presentation-template
 
```

Comment:
@mmore500 since we will have a Docker and Singularity container, and since users do have preference, how about an entrypoint for each? We can't assume that singularity (or docker) are installed, so the user should be asked to choose if it's not specified.

Reply:
Hmm, it might be possible to check to see if `singularity` and/or `docker` are available and adjust the default behavior of the `make ${BUILD_DIR}.pdf` accordingly (i.e., use whichever is available, and just default to Singularity if both are available)?? I assume the most important user preferences would be among users who only have one installed.